doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -34,8 +37,7 @@ To render the documentation on your own machine (macOS or Linux):
 #. Enter ``make html-scipyorg``. This uses the `Make build automation tool`_
    to execute the documentation build instructions from the ``Makefile``.
    This can take a while the first time, but subsequent documentation builds
-   are typically much faster. *Note: If you use a virtual environment for
-   development, activate it first.*
```

Comment:
This needs to be done before installing Sphinx and matplotlib, and step 0 now states "Ensure that you have a working SciPy :ref:`dev-env` active", so I think this should be removed.

If we want to make things like this more explicit (e.g. add instructions for testing whether `import scipy; scipy.__version__` shows the development version number from any arbitrary location), we should add a troubleshooting section. I think this is a good idea for this and other contributor guides. That way the main text can be concise, providing a rather minimal set of instructions that should work for everyone if followed carefully, and common issues can be listed separately for those who run into trouble.